### ISSUE TITLE
LATEST release having trouble with recall mode

### ISSUE THREAD

**Issue by zifeishan** [HazyResearch/mindbender#24]
When using the v0.2.0 for recall mode it goes on well, but it seems that LATEST will break the normal functionality of recall mode. One cannot navigate through words in sentences.

**Comment by zifeishan** [comment]
@netj It seems that this is not fixed yet... Would you debug the recall mode for the latest release? Thanks!

**Comment by zifeishan** [comment]
When I try the LATEST release from my local Mac it shows:

```
Starting Mindtagger for all tasks under /Users/zifei/repos/clearcut/insurance/extractors/labeling/...
Parsing task configuration: label_documents/mindtagger.conf
Exception in thread "main" java.lang.UnsupportedClassVersionError: com/typesafe/config/ConfigMergeable : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at HOCON2JSON.main(hocon2json.scala)
```

and my java version is 1.7.0_67.

**Comment by zifeishan** [comment]
@netj And though it runs on my Linux environment, now I cannot see the tags panel. See below:

I cannot see the dropbox to select the tags.

![image](https://cloud.githubusercontent.com/assets/1811701/8913080/a002ecfa-344b-11e5-9ebf-26f5a450411d.png)

Best,
Zifei

**Comment by netj** [comment]
It seems the Mac binary has been built with Java 8. Will address this. 

The recall mode template syntax has changed a bit since 0.2.x. Could you give a check at the examples?

**Comment by netj** [comment]
@zifeishan Should be fixed by 5d9359e968117380a2e8a50b62ee8b21adbeccbd.  The LATEST binaries will be updated shortly.  It turns out the HOCON library version I used was for Java 8 only.  Can you kindly close this after confirming?

**Comment by zifeishan** [comment]
It works! Thanks!
